crates/vault-registry/src/lib.rs

```diff
@@ -528,6 +543,10 @@ pub mod pallet {
             vault_id: DefaultVaultId<T>,
             banned_until: T::BlockNumber,
         },
+        ClientRelease {
+            version: [u32; 3],
+            checksum: [u8; 32],
```

Comment:
```suggestion
            uri: Vec<u8>,
            code_hash: T::Hash,
```

Reply:
Or if you want to use your `ClientRelease` struct just embed that directly in the event.